cmd/root.go

```diff
@@ -23,6 +23,15 @@ func AddCommands(cmd *cobra.Command) {
 	cmd.PersistentFlags().StringP("preset", "p", "",
 		fmt.Sprintf("preset overwrites default values of the config. options %+s", presets.Options()))
 
+	cmd.PersistentFlags().Bool("recover-from-default-dir", false,
+		"allow nodes to recover from the default recovery dir. only used in systests.")
+
+	/** ======================== Checkpoint Flags ========================== **/
+	cmd.PersistentFlags().StringVarP(&cfg.Recovery.Uri,
```

Comment:
i think it is better to prefix parameters from the same group with unique name. e.g --recovery-uri and --recovery-layer